deps.edn

```diff
@@ -18,7 +18,11 @@
 
 
   :main-dev
-  {:main-opts ["-e" "(require 'genegraph.main)"
+  {:jvm-opts ["-Xmx256m"
+              "-XX:NativeMemoryTracking=detail"]
```

Comment:
Curious about why you choose this option.

Reply:
I thought `NativeMemoryTracking` might give some visibility into native memory allocations being done by Rocks DB, but I don't think it does. I'll remove that option.

Reply:
https://docs.oracle.com/en/java/javase/20/vm/native-memory-tracking.html